build.gradle

```diff
@@ -56,7 +56,6 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
-        jcenter()
```

Comment:
Huh. What was that used for previously?

Reply:
JCenter from Bintray was also a huge Maven repo but it is not public anymore.